core/src/test/scala/integration/kafka/api/EndToEndAuthorizationTest.scala

```diff
@@ -276,7 +276,7 @@ abstract class EndToEndAuthorizationTest extends IntegrationTestHarness with Sas
     * ACL set.
     */
   @Test
-  def testNoConsumeWithDescribeAclViaAssign {
+  def testNoConsumeWithDescribeAclViaAssign(): Unit = {
```

Comment:
nit: comment above has weird alignment. Maybe we can just remove it since the test name seems clear enough?